auth-web/src/components/auth/account-settings/transaction/TransactionsDataTable.vue

```diff
@@ -194,7 +195,10 @@ export default defineComponent({
       }
     })
 
-    const displayDate = (val: Date) => CommonUtils.formatDisplayDate(val, 'MMMM DD, YYYY')
+    const displayDate = (val: Date) => {
+      const targetDate = val ? moment.utc(val).toDate() : val
```

Comment:
I think you need to check if 'val' has a timezone already not if it exists or not here? Now if I used this fn with a date that contained a non utc timezone it would give me the wrong time I think

Reply:
Yup, just being lazy i'll fix it

Reply:
I just used the `moment-timezone` library, makes managing dates and times way easier